packages/ia-components/package.json

```diff
@@ -4,12 +4,14 @@
     "source": true,
     "license": "AGPL-3.0-only",
     "scripts": {
+        "postversion": "node scripts/postversion.js",
```

Comment:
SUGGESTION: This is a "post version" action, where "post" is the verb, right? Seems like our convention is to separate words with hyphens, so maybe "post-version"? Or "version:post"?

Reply:
To have it automatically run after you run the "version" script, you need to follow the NPM naming conventions. https://docs.npmjs.com/misc/scripts